### ISSUE TITLE
hardcoded 26657

### ISSUE THREAD

**Issue by tac0turtle** [nomic-io/nomic#88]
It seems 26657 is hardcoded in the code, users are reporting not being able the port as its listed in the config.toml
```
#[cfg(feature = "full")]
pub fn app_client() -> TendermintClient<app::App> {
  TendermintClient::new("http://localhost:26657").unwrap()
}
```
.

**Comment by Validatus** [comment]
What works is **curl http://127.0.0.1:26657/block** whatever port is configured in the config.toml. 
We have seen this *issue* on multiple nomic nodes. The config.toml values are not *respected*.  

![image](https://user-images.githubusercontent.com/92102712/152060387-1f85431b-656f-41d1-929d-b60f6ef6695d.png)

**Comment by Validatus** [comment]
It seems like the tendermint node is initialized in a way that ignores the config parameters by overwriting them here:
https://github.com/nomic-io/orga/blob/71d146375f7ac160d3dad79bed51da22095b1869/src/abci/node.rs#L44-L71

**Comment by SpacePepeLabs** [comment]
Any update on this? 

Thanks

**Comment by mappum** [comment]
Addressed in `fa1a1911e692e4cd36bba9419ee81ad678d67715` - this will finally be resolved in the next release.

**Comment by Kwaskoff** [comment]
or not =)

nomic start --state-sync-rpc "https://nomic-rpc.polkachu.com:443,https://nomic-rpc.polkachu.com:443"
 2025-01-06T03:06:48.644Z INFO  nomic > nomic v9.0.0
 2025-01-06T03:06:48.644Z INFO  nomic > Configured for network Mainnet
 2025-01-06T03:06:48.645Z INFO  nomic > Starting legacy node... (Command {
    program: "/home/thin/.nomic-stakenet-3/bin/nomic-8.0.1",
    args: [
        "/home/thin/.nomic-stakenet-3/bin/nomic-8.0.1",
        "start",
        "--signal-version",
        "0c",
        "--state-sync-rpc",
        "https://nomic-rpc.polkachu.com:443,https://nomic-rpc.polkachu.com:443",
    ],
    create_pidfd: false,
})
 2025-01-06T03:06:48.664Z INFO  nomic > nomic v8.0.1
 2025-01-06T03:06:48.664Z INFO  nomic > Configured for network Mainnet
 2025-01-06T03:06:48.665Z WARN  nomic > Could not find a legacy binary match, but node is uninitialized, continuing...

███╗   ██╗  ██████╗  ███╗   ███╗ ██╗  ██████╗
████╗  ██║ ██╔═══██╗ ████╗ ████║ ██║ ██╔════╝
██╔██╗ ██║ ██║   ██║ ██╔████╔██║ ██║ ██║
██║╚██╗██║ ██║   ██║ ██║╚██╔╝██║ ██║ ██║
██║ ╚████║ ╚██████╔╝ ██║ ╚═╝ ██║ ██║ ╚██████╗
╚═╝  ╚═══╝  ╚═════╝  ╚═╝     ╚═╝ ╚═╝  ╚═════╝

Version 8.0.1


 2025-01-06T03:06:48.665Z INFO  nomic > Initializing node at /home/thin/.nomic-stakenet-3...
 2025-01-06T03:06:48.732Z INFO  nomic > Getting bootstrap state for Tendermint light client...
 2025-01-06T03:06:48.732Z ERROR nomic > panicked at src/bin/nomic.rs:602:52:
Could not create tendermint RPC client: parse error

Caused by:
    **invalid port number**

Location:
    /home/thin/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flex-error-0.4.4/src/tracer_impl/eyre.rs:10:9
 2025-01-06T03:06:48.737Z ERROR nomic > Legacy node exited unexpectedly
